__tests__/comment.test.js

```diff
@@ -0,0 +1,103 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import sinon from 'sinon';
+import models from '../database/models';
+
+import app from '../index';
+
+chai.use(chaiHttp);
+
+const url = '/api/v1/articles';
+const { expect } = chai;
+let validToken;
+
+describe('Testing comment endpoints', () => {
+  before((done) => {
+    const user = {
+      email: 'dd@test.com',
+      password: 'PasswoRD123__',
+    };
+    chai
+      .request(app)
+      .post('/api/v1/auth/signin')
+      .send(user)
+      .end((err, res) => {
+        const { token } = res.body.user;
+        validToken = token;
+        done();
+      });
+  });
+
+  it('should not post a comment on an article that doesnt exist', (done) => {
+    chai
+      .request(app)
+      .post(`${url}/slug-of-some-article-that-doesnt-exit-in-db/comments`)
+      .set('Authorization', `Bearer ${validToken}`)
+      .send({ body: 'sample comment body' })
+      .end((err, res) => {
+        expect(res.status).to.eql(404);
+        expect(res.body).to.have.property('errors');
+        expect(res.body.errors).to.have.property('message');
+        expect(res.body.errors.message).to.eql('Resource not found');
+        done();
+      });
+  });
+  it('should not post a comment on an article that is not yet published', (done) => {
+    chai
+      .request(app)
+      .post(`${url}/draft-article-slug/comments`)
+      .set('Authorization', `Bearer ${validToken}`)
+      .send({ body: 'sample comment body' })
+      .end((err, res) => {
+        expect(res.status).to.eql(405);
+        expect(res.body).to.have.property('errors');
+        expect(res.body.errors).to.eql('cannot comment on a draft article');
+        done();
+      });
+  });
+  it('should not post a comment with no message in body', (done) => {
+    chai
+      .request(app)
+      .post(`${url}/some-slug/comments`)
+      .set('Authorization', `Bearer ${validToken}`)
+      .send({ body: '' })
+      .end((err, res) => {
+        expect(res.status).to.eql(400);
+        expect(res.body).to.have.property('errors');
+        expect(res.body.errors.body).to.eql('comment body is required');
+        done();
+      });
+  });
+  it('logged in user should post a comment successfully', (done) => {
```

Comment:
Rephrase this statement to form a coherent sentence with `describe`'s "Testing comment endpoints".